src/main/java/org/opensearch/neuralsearch/query/NeuralSparseQueryBuilder.java

```diff
@@ -111,7 +111,7 @@ public static void initialize(MLCommonsClientAccessor mlClient) {
     private NeuralSparseQueryTwoPhaseInfo neuralSparseQueryTwoPhaseInfo = new NeuralSparseQueryTwoPhaseInfo();
 
     private static final Version MINIMAL_SUPPORTED_VERSION_DEFAULT_MODEL_ID = Version.V_2_13_0;
-    private static final Version MINIMAL_SUPPORTED_VERSION_ANALYZER = Version.V_3_0_0;
+    private static final Version MINIMAL_SUPPORTED_VERSION_ANALYZER = Version.V_3_1_0;
```

Comment:
We should make this configurable to pick from bwc version otherwise we need to update this after every version released

Reply:
It is not. This should have been V3.1.0 from the start. The feature is not released in V3.0.0